src/saturn_engine/worker/topics/rabbitmq.py

```diff
@@ -308,3 +315,39 @@ def _deserialize(
             return deserialized
 
         return fromdict(json.loads(message.body.decode()), TopicMessage)
+
+
+class RabbitMQDeadLetterTopic(RabbitMQTopic):
+    class TopicServices:
+        rabbitmq: RabbitMQService
+
+    def __init__(
+        self, options: RabbitMQTopic.Options, services: Services, **kwargs: object
```

Comment:
Since we inherit from `RabbitMQTopic.Options` we have the deat_letter_queue parameters which is not supported here... Maybe we should add a warning if the param is set.